client/components/domains/domain-product-price/index.jsx

```diff
@@ -87,11 +87,50 @@ class DomainProductPrice extends React.Component {
 		return <div className="domain-product-price__price">{ priceText }</div>;
 	}
 
+	renderReskinFreeWithPlanText() {
+		const { isMappingProduct, translate } = this.props;
+
+		const domainPriceElement = ( message ) => (
+			<div className="domain-product-price__free-text">{ message }</div>
+		);
+
+		if ( isMappingProduct ) {
+			return domainPriceElement( translate( 'Included in paid plans' ) );
+		}
+
+		const message = translate( '{{span}}Free for the first year{{/span}}', {
+			components: { span: <span className="domain-product-price__free-price" /> },
+		} );
+
+		return domainPriceElement( message );
+	}
+
+	renderReskinDomainPrice() {
+		const priceText = this.props.translate( '%(cost)s/year', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( '%(cost)s /year' )` *ES Score: 7*